modules/services/utils/src/main/java/au/com/shiftyjelly/pocketcasts/utils/config/FirebaseConfig.kt

```diff
@@ -20,6 +22,7 @@ object FirebaseConfig {
         CLOUD_STORAGE_LIMIT to 10L,
         SLEEP_TIMER_DEVICE_SHAKE_THRESHOLD to 30L,
         REFRESH_PODCASTS_BATCH_SIZE to 200L,
+        EXOPLAYER_CACHE_SIZE_IN_MB to if (BuildConfig.DEBUG) 100L else 0L,
```

Comment:
Not sure if it is needed.

I explicitly added a default value for debug build here as the feature may be enabled even when Firebase config could not be fetched, in this case, we do not want the cache size to be 0.